packages/mobile/src/verify/requestAttestations.ts

```diff
@@ -113,12 +112,9 @@ export function* requestAttestationsSaga() {
       feeless: shouldUseKomenci,
     })
     yield put(revealAttestations())
-  } catch (e) {
-    if (shouldUseKomenci) {
-      yield put(startKomenciSession())
-    } else {
-      yield put(fail(e.message))
-    }
+  } catch (error) {
```

Comment:
nit: remove the try/catch here?  or is there a specific reason to have the `catch (error) { throw error }` pattern?

Reply:
No reason, removed